### DISCUSSION TITLE
Where can I find a benefits list to use fabric gateway sdk?

### DISCUSSION THREAD

**Discussion by SamYuan1990** [hyperledger/fabric-gateway#177] [Q&A]
Hi guys,

I know fabric gateway sdk is basing on fabric sdk.
May I know where can I find a list for benefits using fabric gateway sdk instead of fabric sdk?
it seems I can't find this kind of information from link below
https://hyperledger.github.io/fabric-gateway/

Thanks and Regards
Sam

**Comment by bestbeforetoday** [comment]
There is information in the RFC:

- https://hyperledger.github.io/fabric-rfcs/text/0000-fabric-gateway.html

There is also a summary of the motivation and benefits in the first few minutes of this contributor call:

- https://wiki.hyperledger.org/download/attachments/41590443/20210203_contributors_meeting.mp4?api=v2

**Reply by SamYuan1990** [reply]
https://hyperledger.github.io/fabric-rfcs/text/0000-fabric-gateway.html#reference-level-explanation

the gateway sdk sounds like targeting on simplify client app works by using gateway sdk?

**Comment by davidkhala** [comment]
Quoted from above, I believe these are the bullet points

- This will remove much of the transaction submission logic from the client application which will simplify the maintenance of the SDKs. 
- It will also simplify the administrative overhead of running a Fabric network because client applications will be able to connect and submit transactions via a single network port rather than the current situation where ports have to be opened to multiple peers across potentially multiple organisations.
